### PR TITLE
Update facebook-ads.md

### PR DESCRIPTION
**Pull request by MatthewJamesBoyle**
The instructions are currently incorrect and perhaps a little unclear. This PR resolves that.

### CODE REVIEW

**Comment by brentvatne** [comment]
cc @satya164

File: versions/v22.0.0/sdk/facebook-ads.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ You need to create a placement ID to display ads. Follow steps 1 and 3 from the
...
 
-When using Facebook Ads in development, you'll need to register your device to be able to show ads. You can add the following at the top of your file to register your device:
-
+When using Facebook Ads in development, you'll need to register your device to be able to show ads. Start by adding the following at the top of your App.js (or equivalent) file:
 ```js
 import { FacebookAds } from 'expo';
 
-FacebookAds.AdSettings.addTestDevice(AdSettings.currentDeviceHash);
 ```
+and then in componentWillMount()
```
Comment (satya164) [inline versions/v22.0.0/sdk/facebook-ads.md]: You don't need to do it in componentWillMount

Comment (satya164) [reply versions/v22.0.0/sdk/facebook-ads.md]: It's unnecessary to do it in every `componentWillMount` where the component has an ad. And the instructions are not incorrect per se.

**Comment by MatthewJamesBoyle** [comment]
The current instructions are incorrect, they will lead to errors. If you want me to suggest to do it outside of ComponentwillMount I can but I believe it's the correct thing to do.

**Comment by satya164** [comment]
> The current instructions are incorrect, they will lead to errors. 

Can you tell what errors?

> If you want me to suggest to do it outside of ComponentwillMount I can

That's what's the instructions say right now.

>  but I believe it's the correct thing to do.

Why? Ad settings are global and not tied to a specific component, so I don't see why they need to be done in a component lifecycle.

**Comment by abi** [comment]
Closing due to lack of response, and current instructions seem correct per @satya164

**Pull request closed without merging**